DependencyInjection/Configuration.php

```diff
@@ -0,0 +1,43 @@
+<?php
+
+/*
+ * This file is part of the Symfony CMF package.
+ *
+ * (c) 2011-2014 Symfony CMF
+ *
+ * For the full copyright and license information, please view the LICENSE
+ * file that was distributed with this source code.
+ */
+
+namespace Symfony\Cmf\Bundle\ResourceRestBundle\DependencyInjection;
+
+use Symfony\Component\Config\Definition\ConfigurationInterface;
+use Symfony\Component\Config\Definition\Builder\TreeBuilder;
+
+class Configuration implements ConfigurationInterface
+{
+    /**
+     * Returns the config tree builder.
+     *
+     * @return TreeBuilder
+     */
+    public function getConfigTreeBuilder()
+    {
+        $treeBuilder = new TreeBuilder();
+        $treeBuilder->root('cmf_resource_rest')
+            ->children()
+                ->arrayNode('payload_alias_map')
```

Comment:
Yaml would be the same. The `->useAttributeAsKey` will transform the `alias` attribute in XML to the key of the array. In Symfony XML, prototyped arrays look like this:

````xml
<alias id="article" type="..." />
<alias id="other_alias" type="..." />
````
````yaml
alias_map:
    article: { type: '...' }
    other_alias: { type: '...' }
````
````php
$root
    ->fixXmlConfig('alias', 'alias_map')
    ->children()
        ->arrayNode('alias_map')
            ->useAttributeAsKey('id')
            ->prototype('array')
                ->children()
                    ->scalarNode('type')->end()
                ->end()
            ->end()
        ->end() // alias_map
    ->end();
````